arxivdigest/frontend/database/general.py

```diff
@@ -92,16 +92,15 @@ def insertUser(user):
     return id
 
 
-def insertSystem(system):
+def insertSystem(system_name, user_id):
     """Inserts a new system into the database, name will be used as Name for the system,
     and using uuid a random API-key is generated. Returns None if successfull and an error if not."""
```

Comment:
what is uuid?

Reply:
Universally unique identifiers. Used to get unique api keys